src/EditorFeatures/CSharpTest/CodeActions/ExtractMethod/ExtractMethodTests.cs

```diff
@@ -443,9 +443,7 @@ private static void NewMethod(out Construct obj1, out Construct obj2, out Constr
             await TestAsync(
 @"class Program { static void Main ( string [ ] args ) { [| (int, int) x = (1, 2); |]  System . Console . WriteLine ( x.Item1 ); } } " + TestResources.NetFX.ValueTuple.tuplelib_cs,
 @"class Program { static void Main ( string [ ] args ) { (int, int) x = {|Rename:NewMethod|}(); System.Console.WriteLine(x.Item1); } private static (int, int) NewMethod() { return (1, 2); } }" + TestResources.NetFX.ValueTuple.tuplelib_cs,
-index: 0,
-parseOptions: TestOptions.Regular,
-withScriptOption: true);
```

Comment:
Yes.  The pattern for IDE tests is that the least amount of parameters does the broadest amount of testing.  You only need ot add parameters when you're limiting the testing (because the test doesn't apply in some scenario).